templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/terraform/windowsvm.tf

```diff
@@ -95,7 +95,7 @@ data "template_file" "vm_config" {
     SharedStorageAccess = tobool(var.shared_storage_access) ? 1 : 0
     StorageAccountName  = data.azurerm_storage_account.stg.name
     StorageAccountKey   = data.azurerm_storage_account.stg.primary_access_key
-    FileShareName       = data.azurerm_storage_share.shared_storage.name
+    FileShareName       = var.shared_storage_access ? data.azurerm_storage_share.shared_storage[0].name : ""
```

Comment:
With these changes, the user resource can be deployed if the shared_storage_access variable is set to False and there is no shared storage in the workspace.

Reply:
Should this option also be visible in the UI?

Reply:
I made the change but not actually sure we will need it long term, since we would have a separate user resource template for the review VMs anyway where we can just set it to false by default.